presto-spi/src/main/java/com/facebook/presto/spi/type/FloatType.java

```diff
@@ -35,13 +35,13 @@ private FloatType()
     @Override
     public boolean isComparable()
     {
-        return false;
+        return true;
```

Comment:
I'm fine with squashing commits. I just wanted to it as much incremental as possible, for the sake of readability of PR.